### ISSUE TITLE
OCaml 5.3 support?

### ISSUE THREAD

**Issue by chetmurthy** [alainfrisch/ppx_tools#94]
I was trying out test compatibility with `pa_ppx` and noticed that `ppx_tools` isn't available for OCaml 5.3.  I cloned and tried to build, and got a build failure.  Figured I should report it.  It doesn't get in the way of anything in the Camlp5 ecosystem (of course) but I do like to maintain test-compatibility with the official PPX infrastructure.

**Comment by kit-ty-kate** [comment]
I stopped maintaining ppx_tools after the release of OCaml 5.0 as i wasn't seeing any packages that still used it and i believed it wasn't worth maintaining anymore given 5.1 seemed to have non-trivial changes if my memory serves well.

If anyone has patches they'd like to contribute i'm happy to review and merge them, but i'm honestly not sure it's worth pursuing. However if it sounds like too much of a feature loss for you i'll reconsider. What do you use ppx_tools for? Is it just for compatibility testing? In which case, what kind of compatibility are we talking about?

**Comment by chetmurthy** [comment]
oh hm, perhaps I was mistaken.  It was a while ago when I got this error-message, and today (after a couple of opam upgrade cycles in the weeks since my first post) it's gone.  Sorry, I no longer have any idea which package dependency was complaining about it.

In any case, it was for some standard PPX-based package which I depended-upon in order to do testing, so if there was such a dependency, the author fixed it already.  I'll close this issue.
